app.json

```
@@ -6,6 +6,16 @@
   "repository": "https://github.com/badges/shields",
   "logo": "http://shields.io/favicon.png",
   "env": {
+    "CYPRESS_INSTALL_BINARY": {
+      "description": "Disable the cypress binary installation",
+      "value": 0,
```

```suggestion
      "value": "0",
```

Not sure if this will help, but it's worth a try!